modular_skyrat/modules/aesthetics/clothing/clothing.dm

```diff
@@ -125,6 +125,14 @@
 	icon = 'icons/obj/clothing/under/syndicate.dmi'
 	worn_icon = 'icons/mob/clothing/under/syndicate.dmi'
 
+/*
+*	NINJA
+*/
+
+/obj/item/clothing/under/syndicate/ninja
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/uniform.dmi'
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/uniforms.dmi'
+
```

Comment:
this is already inherited by under/syndicate itself (lines 120/122), unless TG specified them otherwise